.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,20 +13,20 @@
 -->
 
 This PR contains:
-
 - [ ] bugfix
 - [ ] feature
-- [ ] refactor
-- [ ] tests
+- [ ] refactoring
 - [ ] documentation
-- [ ] metadata
-
-### Breaking Changes?
+- [ ] other
 
+Are tests included?
```

Comment:
There are valid reasons not to include tests. CI updates, config updates, README updates, etc. I understand the intent here, but it's not accurate.